Model/Settings/AccountSettings.php

```diff
@@ -32,6 +32,26 @@ class AccountSettings extends BaseModel
         'carrier_configurations',
     ];
```

Comment:
Can you remove this array and remove the attribute definitions in the doc block?

Reply:
Done